CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Note: This CHANGELOG is only for the monitoring team to track all monitoring related changes. Please see OpenShift release notes for official changes.
 
+## 4.11
+- [#1587](https://github.com/openshift/cluster-monitoring-operator/pull/1589) Added support for full-path query log file
```

Comment:
We can omit the changelog entry because the Bugzilla doc text will take care of it.

Reply:
Done ✅